vnext/ReactWindows-Desktop.sln

```diff
@@ -169,12 +169,12 @@ Global
 		{88BAB0FA-E1AC-4DA7-A30C-F91702A8EADB}.Release|x64.Build.0 = Release|x64
 		{88BAB0FA-E1AC-4DA7-A30C-F91702A8EADB}.Release|x86.ActiveCfg = Release|Win32
 		{88BAB0FA-E1AC-4DA7-A30C-F91702A8EADB}.Release|x86.Build.0 = Release|Win32
-		{96CD24DC-91C2-480A-BC26-EE2250DA80D7}.Debug|ARM64.ActiveCfg = Debug|ARM64
+		{96CD24DC-91C2-480A-BC26-EE2250DA80D7}.Debug|ARM64.ActiveCfg = Debug|Win32
 		{96CD24DC-91C2-480A-BC26-EE2250DA80D7}.Debug|x64.ActiveCfg = Debug|x64
 		{96CD24DC-91C2-480A-BC26-EE2250DA80D7}.Debug|x64.Build.0 = Debug|x64
 		{96CD24DC-91C2-480A-BC26-EE2250DA80D7}.Debug|x86.ActiveCfg = Debug|Win32
 		{96CD24DC-91C2-480A-BC26-EE2250DA80D7}.Debug|x86.Build.0 = Debug|Win32
-		{96CD24DC-91C2-480A-BC26-EE2250DA80D7}.Release|ARM64.ActiveCfg = Release|ARM64
+		{96CD24DC-91C2-480A-BC26-EE2250DA80D7}.Release|ARM64.ActiveCfg = Release|Win32
```

Comment:
Just checking if this is intentional, as ARM64 is mapped to Win32, of if this is just VS having a bad day :P

Reply:
note: no vcxproj change suggests this should have happened...

Reply:
Hmm.. I didn't do anything manually.  This was VS.  These appear to be the test projects that got switched, which I dont think build in arm64.  So this is probably correct.